frontend/lib/src/components/elements/ExceptionElement/ExceptionElement.tsx

```diff
@@ -136,6 +142,13 @@ function ExceptionElement({
           ) : null}
           {isLocalhost() && (
             <StyledExceptionLinks>
+              <StyledExceptionCopyButton
+                onClick={() => {
+                  navigator.clipboard.writeText(formattedExceptionFull)
+                }}
+              >
```

Comment:
So some secure contexts, copying will throw an exception, which we would like to prevent.

Is it possible to do the following:

Check for permissions for clipboard-write

```javascript
await window.navigator.permissions.query({ name: permission });
```

We can then only display the CopyButton if it's granted? What do you think? If you like it, bonus points if you modify our other copy button for codeblocks https://github.com/streamlit/streamlit/blob/develop/frontend/lib/src/components/elements/CodeBlock/StreamlitSyntaxHighlighter.tsx#L101

Minor but we also like writing the full window. for clarity this is a global variable.